### ISSUE TITLE
Enable writing of markers.

### ISSUE THREAD

**Issue by felsweg** [ruuda/hound#31]
For now, I haven't found a method to write a marker to a wave file at a specific position. Markers are place din the "format "data section in the "fmt" subchunk. See [here](https://docs.rs/riff-wave/0.1.2/riff_wave/) for more details.

**Comment by ruuda** [comment]
Could you elaborate? The only reference to “marker” on the linked page is

> 65534 | A special marker value, indicating that this is an "extended" wave file.

**Comment by felsweg** [comment]
Yes. To be precise, I am looking for a method to place a "cue" point inside a wave, that is displayable through some audio software like Reaper. The "cue" chunk is explained [here](https://sites.google.com/site/musicgapi/technical-documents/wav-file-format#cue) in more detail.

**Comment by eatspaint** [comment]
Any progress on a release with this? It would be nice to be able to use hound to write cues/markers!

**Comment by felsweg** [comment]
In case it might help you, there is a [workaround](https://stackoverflow.com/a/53177247/8725011) posted on stackoverflow to have cue points being written to wav files.

**Comment by ruuda** [comment]
At the moment there is still no high-level API for writing cue points, the best you can do at this time is to write the chunk manually.

**Comment by catilac** [comment]
I'm writing some audio software that deals with CUE points, so I may make an attempt at a high level API and will put up a PR.
